acceptance-tests/src/main/java/io/blueocean/ath/model/Folder.java

```diff
@@ -1,33 +1,30 @@
 package io.blueocean.ath.model;
 
-import com.google.common.base.Joiner;
-import com.google.common.collect.ImmutableList;
+import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
-import java.util.stream.Collectors;
-import java.util.stream.Stream;
 
 public class Folder {
     private List<String> folders;
 
     public Folder(String ... folders) {
-        this.folders = ImmutableList.copyOf(folders);
+        this.folders = Arrays.asList(folders);
```

Comment:
I don't know if it matters but if an array is passed change to the returned list will "write back" to the array.
https://docs.oracle.com/javase/8/docs/api/java/util/Arrays.html#asList-T...-

 Not the same behavior as `copyOf`.

Reply:
well not a big deal here (it's only used for testing)